tests/unit/s2n_security_policies_test.c

```diff
@@ -67,10 +67,10 @@ int main(int argc, char **argv)
              * TLS 1.3 Ciphers */
             {
                 struct s2n_cipher_suite tls_13_ciphers[] = { s2n_tls13_aes_128_gcm_sha256,
-                                                             s2n_tls13_aes_256_gcm_sha384,
-                                                             s2n_tls13_chacha20_poly1305_sha256 };
+                    s2n_tls13_aes_256_gcm_sha384,
+                    s2n_tls13_chacha20_poly1305_sha256 };
```

Comment:
Can you manually clean up the lists like this in this file? They're going to be more readable if all the cipher suites are aligned.

Like:
```
                struct s2n_cipher_suite tls_13_ciphers[] = {
                    s2n_tls13_aes_128_gcm_sha256,
                    s2n_tls13_aes_256_gcm_sha384,
                    s2n_tls13_chacha20_poly1305_sha256
                };
```
The other list is on line 130